### DISCUSSION TITLE
Feature request - Notification and move plots from destination drive to network drive

### DISCUSSION THREAD

**Discussion by Hindalen** [swar/Swar-Chia-Plot-Manager#866] [Ideas]
Hello.

I was wondering if it could be possible to send a notification if a  "Temp" and/or  "Dest" disk is full. I see that in several occasions plot-manager tries to copy finished plot to a dest drive which is full, maybe because to plots are transferred at the same time, and when it started there was space for this one plot, but not both at the same time. So what happens is that it just stops and I dont get any notifaction that a plot took longer than excepected cause of full destination disk. So is there any way of getting notification if a destinations disk or temp disk is full, and also if there are plots taking longer than usual, for example double/tripple the amount of normal plot creation time.

Second I would like a feature to move plots from a destination drive to a network drive say for example after 6-8 plots is moved to a destination drive, then I need to move them to a farm folder on the harvester which is on another machine. I got a lot of plotter machines with 2-4 TB of destination disks. It would be really great with a feature like this inside plot-manager.

**Comment by bryantcj89** [comment]
GOOD request!
